sql/moz-fx-data-shared-prod/pocket_dbt_facts/dataset_metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: Pocket DBT Fact Models
```

Comment:
We currently require every dataset that's not `user_facing` to be suffixed with [`_derived` or `_external`](https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html#table-layout-and-naming). If that's a problem for DBT we can specifically configure these datasets but it will require the configure to be special case defined in https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-shared/tf/prod/envs/prod/bigquery-new/namespaces.auto.tfvars.json as well.

Reply:
Updated!